sdk/communication/Azure.Communication.CallAutomation/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0 (Unreleased)
 
 ### Features Added
```

Comment:
this section seems to be empty. are there plans to add this section?

Reply:
yes, at the moment there are no changes (no features added) in ga2 except pointing to ga2 swagger. Some changes are coming for GA2, then we will update changelog, but as of no changes after beta2.

Reply:
Here is the api view diff for 1.1.0 beta vs 1.1.0 for your reference: https://apiview.dev/Assemblies/Review/9793bcba87944c9bbdd4e7de431bbba5?diffRevisionId=9ed898b704c64771bcf62b531f9141af&diffOnly=False&revisionId=004ef647b7cd4cc9af9a03995a6dca34&doc=False

It has no changes because so far, no new changes have been made in ga2. There are some changes coming, which we will doing in a separate PR.